### PR TITLE
use environment for author and book urls

### PR DESCRIPTION
**Pull request by allistergg**

### CODE REVIEW

File: src/app/service/author.service.ts [hunk 1]
```diff
@@ -1,32 +1,37 @@
...
+
 
 @Injectable({ providedIn: "root" })
 export class AuthorService {
   constructor(private http: HttpClient) {}
 
   getAuthors() {
     return this.http
-      .get<any>("http://localhost:3000/admin/authors")
+      .get<any>(BASEURL)
```
Comment (jindanupajit) [inline src/app/service/author.service.ts]: The new url for pagination would be `.get<any>(BASEURL+'/paging')`

File: src/app/service/book.service.ts [hunk 2]
```diff
@@ -12,23 +15,23 @@ export class BookService {
   ) { }
 
   getBooks() {
-    return this.http.get<any>('http://localhost:3000/admin/books')
+    return this.http.get<any>(BASEURL)
```
Comment (jindanupajit) [inline src/app/service/book.service.ts]: The new url for pagination would be .get<any>(BASEURL+'/paging')

File: src/app/service/author.service.ts (hunk 1 above)
Comment (allistergg) [reply src/app/service/author.service.ts]: please see new pull request book author form control pagination

File: src/app/service/book.service.ts (hunk 2 above)
Comment (allistergg) [reply src/app/service/book.service.ts]: please see new pull request book author form control pagination

**Pull request closed without merging**